staubli_val3_driver/README.md

```diff
@@ -25,10 +25,10 @@ contents of the `val3_driver` folder to the controller itself.
 
 ### Clone this repository
 
-Clone branch `val3_server-devel` of [staubli_experimental](https://github.com/ocadotechnology/staubli_experimental):
+Clone branch `val3_server-devel` of [staubli_experimental](https://github.com/ros-industrial/staubli_experimental):
 
 ```shell
-git clone https://github.com/ocadotechnology/staubli_experimental -b val3_server-devel
+git clone https://github.com/ros-industrial/staubli_experimental -b val3_server-devel
```

Comment:
Shouldn't users clone just `indigo-devel`? Afaik there is no `val3_server-devel` branch on `ros-industrial/staubli_experimental`?

And as `indigo-devel` is the default, I think we can just omit the branch entirely.